src/internal/connector/exchange/event_test.go

```diff
@@ -27,7 +31,7 @@ func TestEventSuite(t *testing.T) {
 // can be properly retrieved from a models.Eventable object
 func (suite *EventSuite) TestEventInfo() {
 	initial := time.Now()
-	now := common.FormatTime(initial)
+	now := initial.Format(timeFormat)
```

Comment:
Necessary because of https://github.com/alcionai/corso/blob/d2965e0b09a5f10992da51d12627fbcb1b026768/src/internal/connector/exchange/event.go#L40

Reply:
This interpretation should get added to `common/time.go` parsing, instead of distributing its handling across multiple files.